### ISSUE TITLE
XIVLauncher.PatchInstaller cannot apply patch files for 6.5

### ISSUE THREAD

**Issue by reiichi001** [goatcorp/FFXIVQuickLauncher#1385]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to the game updating and plugins not working correctly.

### What did you do?

This is just to make a public issue to track that zipatch seems to have changed or our current zipatch implementation has a bug or is lacking a feature that SE has decided to start using after having never used it previously.

This affects all XIVLauncher.Common-based launchers including XL WPF, XIVLauncher.Core/xlcore, and XIV on Mac/XOM. 

### Platform

Windows

**Comment by gamingvirtue** [comment]
This remains a bug on the Flatpak version, as the update has yet to be pushed there.

**Comment by Taffer** [comment]
This has already been fixed (nice work, Goats!); in my case, I'm just waiting for the Flatpak version to update. XIV on Mac uses the same code and has already been updated. I assume the builds on the Release page work too.

**Comment by GranPasticcio** [comment]
yup, not working for me either, running on linux with flatpak, v.1.0.4

**Comment by gamingvirtue** [comment]
It shouldn't be much longer, I'd imagine. XIVLauncher.Core was updated earlier, and it just seems the only thing holding it up is an error in the release process. Been keeping a close eye on it all morning, since I'm awaiting the update, too.
